examples/swr-site/pages/docs/data-fetching.en-US.mdx

```diff
@@ -32,7 +32,7 @@ function App () {
 
 <Callout emoji="💡">
   If you are using <strong>Next.js</strong>, you don't need to import this polyfill:<br/>
-  <a target="_blank" rel="noopener" href="https://nextjs.org/blog/next-9-1-7#new-built-in-polyfills-fetch-url-and-objectassign">New Built-In Polyfills: fetch(), URL, and Object.assign</a>
+  [New Built-In Polyfills: fetch(), URL, and Object.assign](https://nextjs.org/blog/next-9-1-7#new-built-in-polyfills-fetch-url-and-objectassign)
```

Comment:
redundant since all links that starts with `https://` will have `target=_blank`
https://github.com/shuding/nextra/blob/dfbe9965eeec39929b1eb67b62600fddc561eae0/packages/nextra-theme-docs/src/misc/theme.tsx#L220